src/ploomber/cloud/api.py

```diff
@@ -175,7 +175,11 @@ def run_detail_print(run_id):
     elif tasks:
         click.echo(Table.from_dicts(tasks))
     else:
-        click.echo('Pipeline up-to-date, no tasks scheduled for this run.')
+        if run['status'] == 'finished':
```

Comment:
What happens here on aborted pipelines?

Reply:
Also can you move those conditions a level above to being part of the switchcase?